src/server/repository/adapter/assessment.ts

```diff
@@ -0,0 +1,14 @@
+import { Objects } from '@utils/objects'
+
+import { Assessment, AssessmentProps } from '@meta/assessment'
+
+interface AssessmentDB {
+  id: number
+  uuid: string
+  props: AssessmentProps
+  meta_cache: AssessmentProps
```

Comment:
meta_cache is of AssessmentMetaCache type 
import { Assessment, AssessmentMetaCache , AssessmentProps } from '@meta/assessment'